test/runner/.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+	"root": true,
+
+	"extends": "../../.eslintrc-node.json",
+
+	"overrides": [
+		{
+			"files": ["**/*"],
+      "globals": {
+        "fetch": false
+      },
+			"parserOptions": {
+        "ecmaVersion": 2022,
+				"sourceType": "module"
```

Comment:
Indentation is off in this file.

Reply:
Ok, this is a good argument for not changing all the JSON indentation, at least not yet. Maybe when we migrate off grunt and use eslint flat config, we could introduce spaces in JSON if we agree.

Reply:
I think I'd prefer tabs in JSON as well for internal code style consistency. Not a strong preference but it feels to me like a better choice here.

Reply:
But yeah, whatever we settle on can be done separately. I mostly care about here that it's at least consistent within a single file. 😉